project.clj

```diff
@@ -1,7 +1,7 @@
 (def schema-tools-version "0.9.0")
 (def schema-version "1.1.3")
 
-(defproject threatgrid/ctim "0.4.3"
+(defproject threatgrid/ctim "0.4.4-SNAPSHOT"
```

Comment:
The change will require a new version.  The alternative is to bump the version directly in master, which I'm fine doing, but I'm also fine doing it in the PR.